dashboard/v2/.bowerrc

```diff
@@ -1,5 +1,5 @@
 {
-  "directory": "dist/lib",
+  "directory": "public/lib",
```

Comment:
All the generated/downloaded files should go to target. Hence there is option of mvn clean. For offline build, one should use mvn install (without clean). Please change to target